Content.Client/Overlays/ShowSecurityIconsSystem.cs

```diff
@@ -74,7 +74,11 @@ private IReadOnlyList<StatusIconPrototype> DecideSecurityIcon(EntityUid uid)
                 result.Add(icon);
         }
 
-        // Add arrest icons here, WYCI.
+        var ev = new GetCriminalIconEvent();
+        RaiseLocalEvent(uid, ref ev);
+
+        if (ev.Icon != null && _prototypeMan.TryIndex<StatusIconPrototype>(ev.Icon.Value.Id, out var criminalIcon))
+            result.Add(criminalIcon);
```

Comment:
Just TryIndex and if it doesn't support nullable then add it.

Reply:
I replaced the event with a TryComp, so icon can't be null anymore.